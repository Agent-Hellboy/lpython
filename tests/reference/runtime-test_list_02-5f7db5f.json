{
    "basename": "runtime-test_list_02-5f7db5f",
    "cmd": "lpython {infile}",
    "infile": "tests/runtime_errors/test_list_02.py",
    "infile_hash": "004ac679732e715b87556ac0d6f0ea649612981999a20dad4e25acd0",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "runtime-test_list_02-5f7db5f.stderr",
    "stderr_hash": "f29a370e7602b2ba55247305a0cd54ef6704ae9eb123ad2c869ebb5f",
    "returncode": 1
}