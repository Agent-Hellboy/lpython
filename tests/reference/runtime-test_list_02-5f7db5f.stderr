ld: warning: object file (test_list_02.out.tmp.o) was built for newer macOS version (14.0) than being linked (13.3)
IndexError: List index is out of range. Index range is (0, 2), but the given index is 3
